src/screens/Ticketing/Ticket/TicketInfo.tsx

```diff
@@ -96,6 +103,49 @@ export const TicketInfoView = (props: TicketInfoViewProps) => {
   );
 };
 
+const WarningMessage = (
+  isError: boolean,
+  remoteTokens?: RemoteToken[],
+  isInspectable?: boolean,
+) => {
+  const {t} = useTranslation();
+  const inspectableToken = findInspectable(remoteTokens);
+  if (isError)
+    return (
+      <ThemeText type="body__secondary">
+        {t(TicketTexts.warning.unableToRetrieveToken)}
+      </ThemeText>
+    );
+  if (remoteTokens && remoteTokens.length === 0)
+    return (
+      <ThemeText type="body__secondary">
+        {t(TicketTexts.warning.noTokenFound)}
+      </ThemeText>
+    );
```

Comment:
An empty `remoteTokens?` list will mae `isError` `true`, so this clause is redundant. Just showing the `unableToRetrieveToken` is good enough, as something has gone wrong.

Reply:
Ohh Oka. I will get rid of this. It was in one of the cases mentioned in the issue, so I thought its a valid scenario :)